src/main/java/ch/njol/skript/expressions/ExprItemOwner.java

```diff
@@ -0,0 +1,79 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Item Owner")
+@Description("The owner of the dropped item.")
+@Examples({
+	"broadcast the item owner of all dropped items"
+})
+@Since("INSERT VERSION")
+public class ExprItemOwner extends SimplePropertyExpression<Entity, OfflinePlayer> {
+
+	static {
+		registerDefault(ExprItemOwner.class, OfflinePlayer.class, "item owner", "entities");
+	}
```

Comment:
Might be worth merging this into `ExprOwner` it seems weird to have multiple similar syntax

Reply:
You can make the patterns separate